sdrf_pipelines/openms/openms.py

```diff
@@ -59,7 +60,7 @@ def _removesuffix(x: str, suffix: str, /) -> str:
 
     raw_bkp = raw
     for current_extension, target_extension in extension_convert_dict.items():
-        if raw.endswith(current_extension):
+        if raw.lower().endswith(current_extension.lower()):
             raw = _removesuffix(raw, current_extension)
             raw += target_extension
             if not any(raw.endswith(x) for x in possible_extension):
```

Comment:
I think you will need to change this,too in case the user uses RAW and does no conversion of it.

Reply:
Here I assume that the final openms experimental design file has the same suffix. Is that right? For example
Example 1: `raw:mzML`: 
1.raw    -> 1.mzML
2.Raw    -> 2.mzML
3.RAW   -> 3.mzML

Example 2: `raw:RAW`:
1.raw -> 1.RAW
2.Raw -> 2.RAW
3. RAW -> 3.RAW

Reply:
And also we assumed only suppoted four format: `"raw", "mzML", "mzml", "d"` https://github.com/bigbio/sdrf-pipelines/blob/d87d22c7b763f290d4cebb7c336a66af205be527/sdrf_pipelines/openms/openms.py#L66-L67

Reply:
If I am not mistaken your example raw:RAW will fail.
If you only allow ".raw" as possible extension you have to add raw:raw as a default conversion to the pipeline which is super weird and not expected.

Reply:
Why not allow the possible extensions case insensitive then?
Would solve a lot of problems.

Reply:
It depends a bit on if downstream tools allow different cases. (e.g. does DIANN allow .D?)

Reply:
Good ideas. I need to test it. Wait a while. Also is ProteomicsLFQ allowed different cases? For example input` .mzML` file, but in the lab file it's `mzml`

Reply:
ProteomicLFQ expects *exact* matches of the full filename of the files that you pass as -in and what is written in the experimental design (including extension, excluding paths).

Reply:
DIA-NN don't support `.D` format. So I suggest the possible extensions is case-sensitive. What do you think?

Reply:
> Is this conversion to openms experimental design even used in the DIA branch of the pipeline?

Yes. It also used in DIA branch https://github.com/bigbio/quantms/blob/b7c9b6a416723c4210effd3889dadce3d9032938/workflows/quantms.nf#L55